ee/query-service/usage/manager.go

```diff
@@ -123,6 +139,18 @@ func (lm *Manager) UploadUsage() {
 
 	zap.S().Info("uploading usage data")
 
+	// Try to get the org name
+	orgName := ""
+	orgNames, err := lm.modelDao.GetOrgs(ctx)
+	if err != nil {
+		zap.S().Errorf("failed to get org data: %v", zap.Error(err))
+	}
+	if len(orgNames) != 1 {
+		zap.S().Errorf("expected one org but got %d orgs", len(orgNames))
+	} else {
+		orgName = orgNames[0].Name
```

Comment:
Will there be non-zero `orgNames` all the time, even if `err` is non-nil?

Reply:
Thanks for noticing, have fixed.